### PR TITLE
Linear regression

### LINKED ISSUE #57: Linear regression model

### PR DESCRIPTION
**Pull request by guilhermefgs**
Close #57

### CODE REVIEW

File: tatu_re/model/linear_regression.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+from sklearn.preprocessing import PolynomialFeatures
+from sklearn.impute import SimpleImputer
+from sklearn.pipeline import Pipeline
+from sklearn.linear_model import LinearRegression
+
+from tatu_re.model.data_processing import IndicatorsTransformer
+
+def linear_regression_pipeline():
+
+    pipe = Pipeline([
```
Comment (guilhermefgs) [inline tatu_re/model/linear_regression.py:12]: Pipeline currently not working

IndexError: single positional indexer is out-of-bounds

File: tatu_re/model/train.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+from tatu_re.model.data_processing import calculate_target
+from tatu_re.model.linear_regression import linear_regression_pipeline
+from tatu_re.utils import get_data
+
+from datetime import datetime
+import pathlib
+import pickle
+
+def train_model(save=True):
```
Comment (caio-freitas) [inline tatu_re/model/train.py:9]: Maybe it's good to add the begin and end dates as parameters to this function

Comment (guilhermefgs) [reply tatu_re/model/train.py:9]: Yes, I created something nice to execute this function. But I don't know yet how to use parameters.

Basically you run `pip install -e .` in the directory, this installs the dependencies but also allows to run a script that calls train-model. Which executes this code.

**Comment by guilhermefgs** [comment]
The model is recommending more "sell" than "buy"s (imbalanced). And the criteria for the amount of buying is not good. Should we consider a negative number of stocks?

**Pull request merged**